src/predicate/isMap.ts

```diff
@@ -16,6 +16,6 @@
  * console.log(isMap(value3)); // false
  */
 
-export function isMap(value: unknown): value is Map<any, any> {
+export function isMap(value?: unknown): value is Map<any, any> {
```

Comment:
```suggestion
export function isMap(value: unknown): value is Map<any, any> {
```

I think our original library has to accept the argument here.

Reply:
I appreciate your confirmation!
Thank you 👍